lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
This is what we have in ShopifyApp `'omniauth-shopify-oauth2', '~> 1.1.11'`
https://github.com/Shopify/shopify_app/blob/master/shopify_app.gemspec#L14